scripts/travis_build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+# Doing jobs according to COMPONENT's value to minimize build times
+# if [ "$JOB" == "TEST" ]; then
```

Comment:
Please delete these lines. Also, since this file contains only one useful line, consider adding it directly to the .travis.yml